IconToggle/index.d.ts

```diff
@@ -2,10 +2,15 @@ import MaterialComponent from '../MaterialComponent';
 import { VNode } from 'preact';
 import { MDCFoundation, MDCComponent, MDCRipple } from '../MaterialComponentsWeb';
 
+declare interface IconToggleState {
+  label?: string;
+  content?: string;
+  cssClass?: string;
+}
+
 declare interface IIconToggleProps extends JSX.HTMLAttributes {
-  disabled?: boolean;
-  'data-toggle-on'?: any;
-  'data-toggle-off'?: any;
+  'data-toggle-on'?: IconToggleState;
+  'data-toggle-off'?: IconToggleState;
 }
 export default class IconToggle extends MaterialComponent<IIconToggleProps, {}> {
```

Comment:
`IconToggleState` is not added to the MaterialComponent.

```TypeScript
export default class IconToggle extends MaterialComponent<IIconToggleProps, IconToggleState> {
```

Reply:
Ah, yes. Confusing name for the type. It's not the State for the component. Just named it after something discussed in the docs, but it's obviously misleading in this context. I'll rename it to something like IconToggleData etc.